notebooks/Tutorial8-Time_series.py

```diff
@@ -0,0 +1,287 @@
+        n_points = int(proportion_to_plot * len(selected_records))
+        selected_indices = np.linspace(0, len(selected_records) - 1, num=n_points)
+
+        x = selected_records[time_variable].iloc[selected_indices]
+        y = selected_records[variable_to_plot].iloc[selected_indices]
+        ax.plot(x, y, color=cmap(id_nb / n_series_to_plot))
+    ax.set_title(f"Series for variable {variable_to_plot}")
+    return fig
+
+
+# -
+
+# and we use it to plot sensor data for some individuals
+
+# ### `sensor1` data
+
+plot = plot_series(
+    df := ts1_df,
+    variable_to_plot="sensor1",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+# ###  `sensor2` data
+
+plot = plot_series(
+    df := ts1_df,
+    variable_to_plot="sensor2",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+# ### `sensor3` data
+
+plot = plot_series(
+    df := ts2_df,
+    variable_to_plot="sensor3",
+    id_variable="id",
+    time_variable="t",
+    proportion_to_plot=1.0,
+    n_series_to_plot=None,
+    figsize=(14, 8),
+)
+plot.show()
+
+# Now that we know what data we are maniplating, we can anonymize it.
+
+# ## Avatars of time series data
+
+# ### Upload data and save datasets
+
+# +
+dataset_vanilla = client.pandas_integration.upload_dataframe(vanilla_df)
+
+ts_dfs = [
+    ts1_df,
+    ts2_df,
+]  # we put all time series dataframes in a list to simplify further operations
+datasets_ts = []
+for ts_df in ts_dfs:
+    datasets_ts.append(client.pandas_integration.upload_dataframe(ts_df))
+# -
+
+# ### Create job and retrieve results
+#
+# This is done in a similar way than for the anonymization of classic tabular data. However, the objects are different. For data containing time series, it is necessary to parameterize a `AvatarizationWithTimeSeriesJobCreate` with parameters of type `AvatarizationWithTimeSeriesParameters`.
+#
+# The attributes to specify in `AvatarizationWithTimeSeriesParameters` are:
+# - `k`
+# - the dataset_id of the vanilla dataset (i.e. the individual-centric dataset containing one line per individual)
+# - a list of `AvatarizationTimeSeriesParameters` containing 1 item per time series dataset.
+#
+#
+# Each `AvatarizationTimeSeriesParameters` specifies the dataset_id of the time series data and the projection parameters. The projection parameters defines how the data will be transformed (aka projected) so that avatars can be generated.
+#
+# There are currently only one type of projection, Functional Principal Component Analysis (FPCA) and so the `projection_parameters` should be a `FPCAParameters`. Additional projectors will be made available to enable better handling of certain types of time series data. Finally, the `nf` attribute defines how many components should be used to represent each time series variable when generating avatars.
```

Comment:
```suggestion
# Supported time series projection is Functional Principal Component Analysis (FPCA) . `nf` is the number of components used to represent each time series variable in the projection.
```

https://docs-internal.octopize.io/docs/process/documentation#docs-best-practices
Prefer short, simple sentences. Stay away from the heavy French way of writing.